robottelo/ssh.py

```diff
@@ -241,6 +241,22 @@ def upload_file(local_file, remote_file, key_filename=None, hostname=None):
             sftp.close()
 
 
+def upload_files(local_dir, remote_dir, file_type='.txt'):
```

Comment:
perhaps i would not implement a whole new function but tried to modify the `upload_file` function instead to also accept lists.
then i would simply iterate over the list in a context of a single session to upload all the files from this list.
That would be more robust and give user the power of pick the files he'd like to upload

Reply:
@rplevka : This is also one of the way to achieve same solution. But i thought about that adding a wrapper function would add more easiness if some one wanted to add new tests based on different file formats.